tests/parm/global_control.nml.IN

```diff
@@ -73,15 +73,15 @@
   nggps_ic = @[NGGPS_IC]
   mountain = @[MOUNTAIN]
   ncep_ic = .false.
-  d_con = 1.
+  d_con = 1
```

Comment:
d_con and consv_te are real numbers, can we keep them real?

Reply:
Done, these have been changed to reals.